packages/labbook/src/components/upload-button.tsx

```diff
@@ -7,12 +7,11 @@ export interface IUploadButtonProps {
   label?:string;
   disabled?: boolean;
   onClick?: (type: string) => void;
-  inDialog?: boolean;
 }
 
 export const UploadButton: React.FC<IUploadButtonProps> = (props) => {
-  const {children, onClick, disabled, inDialog} = props;
-  const classes = classNames(css["upload-button"], {[css.disabled]: disabled, [css["in-dialog"]]: inDialog});
+  const {children, onClick, disabled} = props;
+  const classes = classNames(css["upload-button"], {[css.disabled]: disabled});
```

Comment:
I think this may break the final fix I made for media upload buttons that QA found?  Could you test this with an activity that exports an image to the media library and then enables the media library in the labbook?  You'll need to first select a media library image and then replace it and check the button padding/size.

Reply:
Added back the inDialog css. Don't even remember why I removed it.